src/nodes/ImageView.vue

```diff
@@ -41,6 +41,7 @@
 							<div class="media__wrapper">
 								<img v-show="loaded"
 									:src="imageUrl"
+									:alt="alt"
 									class="image__main"
 									@load="onLoaded">
 								<div class="metadata">
```

Comment:
@juliushaertl maybe we should hide the span below (`class="name"`) for a11y? meaning making it `aria-hidden="true"`. Because it duplicates the alt tag.